### ISSUE TITLE
Does World Locking Tools work with Scene Understanding?

### ISSUE THREAD

**Issue by jackmca** [microsoft/MixedReality-WorldLockingTools-Unity#303]
Just a quick question as mentioned in the title, I was wondering if anyone has tried to use World Locking Tools and Scene Understanding, I couldn't find anything online about this topic, cheers.

**Comment by fieldsJacksonG** [comment]
Yes: World locking tools works by moving the player camera's parent node the same way you might implement teleporting the player to a different part of the map.

The player camera and scene understanding will both still get transform updates from the same coordinate system or XrSpace.
